src/migrate.js

```diff
@@ -39,36 +39,6 @@ function migrateWarn( msg ) {
 	}
 }
 
-function migrateWarnProp( obj, prop, value, msg ) {
```

Comment:
Wow, we don't need this anymore? It's another one I suspect will be back, but might as well remove it. If it came back we wouldn't need the guard for `Object.defineProperty` or the `try/catch` anyway.